### PR TITLE
Parsed Geo Json Geometry can now be converted to Atlas Geometry

### PR DESCRIPTION
**Pull request by yazad3**
### Description:

Parsed Geo Json Geometry can now be converted to Atlas Geometry

| Geo Json Geometry        | Atlas Geometry                                                                        |
| -------------------------|---------------------------------------------------------------------------------------|
| Point                    | `Location`                                                                            |
| MultiPoint               | `List<Location>`                                                                      |
| LineString               | `PolyLine`                                                                            |
| MultiLineString          | `List<PolyLine>`                                                                      |
| Polygon                  | `Polygon`                                                                             |
| MultiPolygon             | `List<Polygon>`                                                                       |
| GeometryCollection       | *N/A* (Non-GeometryCollection children can be converted to the above.)                | 

### Potential Impact:

N/A.

### Unit Test Approach:

Additional tests added to test the conversion.

### Test Results:

Tests successful.

### CODE REVIEW

**Comment by ranek** [comment]
This looks very useful, thanks!

Both `Polygon` and `MultiPolygon` in GeoJSON may have holes or interior rings. How do you plan to deal with these?

**Comment by yazad3** [comment]
@ranek - thanks for the comment. This has been addressed in the code, test cases and documentation.

File: src/main/java/org/openstreetmap/atlas/geography/geojson/parser/README.md [hunk 1]
```diff
@@ -231,6 +231,18 @@ This means that for the auto-mapper to work you need to follow the `JavaBean` st
...
+#### Parsed Geo JSON Geometry to Atlas Geometry mapping
+
+| Geo Json Geometry        | Atlas Geometry                                                                        |
+| -------------------------|---------------------------------------------------------------------------------------|
+| Point                    | `Location`                                                                            |
+| MultiPoint               | `List<Location>`                                                                      |
+| LineString               | `PolyLine`                                                                            |
+| MultiLineString          | `List<PolyLine>`                                                                      |
+| Polygon                  | `Polygon`                                                                             |
+| MultiPolygon             | `List<Polygon>`                                                                       |
```
Comment (matthieun) [inline src/main/java/org/openstreetmap/atlas/geography/geojson/parser/README.md]: Any reason this is not going to Atlas `MultiPolygon`?

Comment (yazad3) [reply src/main/java/org/openstreetmap/atlas/geography/geojson/parser/README.md]: Yes, I had it originally as a MultiPolygon but there is no easy way with Atlas MultiPolygon currently to handle donuts. See comment from Ranek above.

**Comment by lucaspcram** [comment]
@yazad3 Does this properly handle GeoJson polygons? Per @ranek it is possible for a GeoJson polygon to have an inner ring. So shouldn't we have Polygon -> `List<Polygon>` and not just a plain old Atlas `Polygon`?

**Pull request merged**